winthrop_client_python/refresh_token.py

```diff
@@ -0,0 +1,52 @@
+import requests
```

Comment:
Please run `black` against this file to standardize the formatting. otherwise it looks good.